packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -0,0 +1,262 @@
+      src: '/file.html',
+      headers: { Location: '/file' },
+      status: 301,
+    },
+    {
+      src: '/path/to/file.html',
+      headers: { Location: '/path/to/file' },
+      status: 301,
+    },
+    { src: '/file', dest: '/file.html', continue: true },
+    { src: '/path/to/file', dest: '/path/to/file.html', continue: true },
+  ];
+  deepEqual(actual, expected);
+
+  const mustMatch = [
+    ['/file.html'],
+    ['/path/to/file.html'],
+    ['/file'],
+    ['/path/to/file'],
+  ];
+
+  const mustNotMatch = [
+    ['/file2.html', '/afile.html'],
+    ['/path/to/file2.html', '/path/to/file'],
+    ['/file2', '/afile'],
+    ['/path/to/file2', '/file'],
+  ];
+
+  assertRegexMatches(actual, mustMatch, mustNotMatch);
+});
+
+test('convertRedirects', () => {
+  const actual = convertRedirects([
+    { source: '/some/old/path', destination: '/some/new/path' },
+    {
+      source: '/firebase/(.*)',
+      destination: 'https://www.firebase.com',
+      statusCode: 302,
+    },
+    {
+      source: '/projects/:id/:action',
+      destination: '/projects.html',
+    },
+    { source: '/old/:segment/path', destination: '/new/path/:segment' },
+  ]);
+
+  const expected = [
+    {
+      src: '^\\/some\\/old\\/path$',
+      headers: { Location: '/some/new/path' },
+      status: 307,
+    },
+    {
+      src: '^\\/firebase\\/(.*)$',
+      headers: { Location: 'https://www.firebase.com' },
+      status: 302,
+    },
+    {
+      src: '^\\/projects\\/([^\\/]+?)\\/([^\\/]+?)$',
+      headers: { Location: '/projects.html?id=$1&action=$2' },
+      status: 307,
+    },
+    {
+      src: '^\\/old\\/([^\\/]+?)\\/path$',
+      headers: { Location: '/new/path/$1' },
+      status: 307,
+    },
+  ];
+
+  deepEqual(actual, expected);
+
+  const mustMatch = [
+    ['/some/old/path'],
+    ['/firebase/one', '/firebase/2', '/firebase/-', '/firebase/dir/sub'],
+    ['/projects/one/edit', '/projects/two/edit'],
+    ['/old/one/path', '/old/two/path'],
+  ];
+
+  const mustNotMatch = [
+    ['/nope'],
+    ['/fire', '/firebasejumper/two'],
+    ['/projects/edit', '/projects/two/three/delete', '/projects'],
+    ['/old/path', '/old/two/foo', '/old'],
+  ];
+
+  assertRegexMatches(actual, mustMatch, mustNotMatch);
+});
+
+test('convertRewrites', () => {
+  const actual = convertRewrites([
+    { source: '/some/old/path', destination: '/some/new/path' },
+    { source: '/firebase/(.*)', destination: 'https://www.firebase.com' },
+    { source: '/projects/:id/edit', destination: '/projects.html' },
+  ]);
+
+  const expected = [
+    { handle: 'filesystem' },
+    { src: '^\\/some\\/old\\/path$', dest: '/some/new/path', continue: true },
+    {
+      src: '^\\/firebase\\/(.*)$',
+      dest: 'https://www.firebase.com',
+      continue: true,
+    },
+    {
+      src: '^\\/projects\\/([^\\/]+?)\\/edit$',
+      dest: '/projects.html?id=$1',
+      continue: true,
+    },
+  ];
+
+  deepEqual(actual, expected);
+
+  const mustMatch = [
+    [],
```

Comment:
What's the reason for this?

Reply:
The first returned route is `handle:filesystem` so we can't run regex tests on that